mitodl/micromasters
# Implemented UI for financial aid states

#### What are the relevant tickets?
#1115

#### What's this PR do?
Implements the UI for the financial aid card

Note that:
- The 'skip this and pay full price' links are not implemented yet.
- There's a lot of little design changes that aren't made yet in this PR.
- The document send button does not function yet.
- UI tests are omitted for the most part, we will need to manually test this instead.

#### How should this be manually tested?
On a program without financial aid, make sure the card doesn't show up at all.

For a program with financial aid, iterate through the different statuses by updating the `FinancialAid` object in the Django shell. For each state make sure the items on the page match the mockups [here](https://projects.invisionapp.com/share/NB7VM1GSG#/screens/190510429)

#### Screenshots (if appropriate)
